<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Units</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="weightLabel">
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="weightComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="temperatureLabel">
         <property name="text">
          <string>Temperature</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="temperatureComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="volumeLabel">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="volumeComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="gravityLabel">
         <property name="text">
          <string>Gravity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="gravityComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="dateComboBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="colorLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="colorComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="title">
       <string>Formulas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Forumulas</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ibuFormulaLabel">
            <property name="text">
             <string>IBU</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="ibuFormulaComboBox">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="colorFormulaLabel">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="colorFormulaComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>IBU Adjustments</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="mashHopLabel_2">
            <property name="text">
             <string>Mash Hop (%)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="ibuAdjustmentMashHopDoubleSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>200.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="firstWortLabel">
            <property name="text">
             <string>First Wort (%)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="ibuAdjustmentFirstWortDoubleSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>200.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Database Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dbDir">
           <property name="toolTip">
            <string>Where your brewtarget .xml files are.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dbDirBrowse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dbDirDefault">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>173</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_language">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QComboBox" name="comboBox_lang">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;qt&gt;
   &lt;b&gt;Know another language?&lt;/b&gt;
   &lt;br&gt;&lt;br&gt;
   Or, would you like to improve a translation? Help us out and
  &lt;a href=&quot;https://sourceforge.net/sendmessage.php?touser=938941&quot;&gt;
  provide a translation&lt;/a&gt; so that your friends can use brewtarget!
&lt;/qt&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_dbDir</tabstop>
  <tabstop>pushButton_dbDirBrowse</tabstop>
  <tabstop>pushButton_dbDirDefault</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>optionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>optionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
